buildall

```diff
@@ -12,6 +12,49 @@ buster
 unstable
 "
 
+DISTS_WITH_SNAPSHOT="buster"
```

Comment:
If I only want to build, say, "jessie", I'd run `./buildone jessie`.

How can I just run a snapshot build?

Technically I have to run `./buildone "$DIST" "$snapshot_id"`, but I need to compute "snapshot_id" manually.

What if you factor out the new code from `buildall` into a `build_latest_snapshot` script so that we can manually invoke that if we need to?

Usage:
```
$ build_latest_snapshot buster
```

`buildall` will then simply invoke `build_latest_snapshot $DISTS_WITH_SNAPSHOT`

Reply:
I see your point but i need to add something, when having more than 1 dist let's say:

```bash
DISTS_WITH_SNAPSHOT=buster
jessie
```

Using `build_latest_snapshot $DIST` it could be possible to get one `snapshot_id` for `buster` and another different one for `jessie` since debian is doing 4 snapshot id per day and we don't know when.

I could suggest to only factor out the part of the code regarding `snapshot_id` into a new scripts and then we can manually invoke:

```
$ ./buildone buster "$(snapshot_id)"
```

what do you think?

Reply:
I see.

What about this (pseudo code)

```
$ ls
buildone
buildall
build_latest_snapshot
snapshot_id
...
$ cat build_latest_snapshot
...
./buildone $1 $(./snapshot_id)
$ cat buildall
...
for i in $DISTS; do
  ./buildone $i
done
for i in $DISTS_SNAPSHOT; do
  ./build_lastest_snapshot $i
done
...
```

The rationale is that in order to type `./buildone buster "$(./snapshotid)"` you to know how this works, i.e. you need to read some instructions. Instructions tend to get out of date quickly.

Reply:
Ok, done!